gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/auth/idp/impl/IdentityProviderManagerImpl.java

```diff
@@ -139,6 +139,7 @@ public void onEvent(Event<IdentityProviderEvent, Payload> event) {
                     removeIdentityProvider(event.content().getId());
                     gatewayMetricProvider.decrementIdp();
                     break;
+                default: break;
```

Comment:
break in default? 

Can we not use a new switch here?

Reply:
https://sonarcloud.io/organizations/gravitee-io/rules?open=java%3AS131&rule_key=java%3AS131